doc/mkdocs/docs/api/basic_json/operator_eq.md

```diff
@@ -46,7 +56,11 @@ Linear.
 
 !!! note
```

Comment:
Add title "NaN handling".

Reply:
How about "Handling NaN and special values"?

I'm trying to reconcile the explanations about `null` and discarded in the same block. Although, `null` isn't particularly special...
Close enough?

Reply:
What about "Handling NaN and `null`"? Or do we have special values beyond?

(I think `null` is somewhat special)

Reply:
`NaN`, `null`, and discarded.

"Handling of `NaN`, `null`, and discarded values"?

Also, does "handling" accurately capture what's being talked about in this note?

How about:
(1) "Comparison of `NaN`, `null`, and discarded values" or (2) simply "Comparison of special values".

(In the other block(s) we have `NaN` and an explanation about the rewritten candidate operator, which I've split into a separate note.)

Reply:
"Comparison of special values" 👍